openmetadata-ui/src/main/resources/ui/src/components/Persona/PersonaSelectableList/PersonaSelectableList.test.tsx

```diff
@@ -0,0 +1,172 @@
+  }),
+}));
+
+jest.mock('../../common/SelectableList/SelectableList.component', () => ({
+  SelectableList: jest
+    .fn()
+    .mockImplementation(
+      ({
+        children,
+        customTagRenderer,
+        searchPlaceholder,
+        fetchOptions,
+        onUpdate,
+      }) => (
+        <div data-testid="selectable-list">
+          {children}
+          {searchPlaceholder}
+          {customTagRenderer('add-user')}
+          <button
+            data-testid="fetch-options"
+            onClick={() => fetchOptions('username')}>
+            fetchOptions
+          </button>
+          <button data-testid="update" onClick={onUpdate}>
+            Update
+          </button>
+        </div>
+      )
+    ),
+}));
+
+jest.mock('antd', () => ({
+  ...jest.requireActual('antd'),
+
+  Popover: jest.fn().mockImplementation(({ content, children, showArrow }) => {
+    return (
+      <div data-testid="popover">
+        {content}
+        {children}
+        <button data-testid="show-arrow-button" onClick={() => showArrow(true)}>
+          showArrow
+        </button>
+      </div>
+    );
+  }),
+}));
+
+const mockUpdate = jest.fn();
+
+describe('PersonaSelectableList', () => {
+  it('should render Persona Selectable List', () => {
+    render(
+      <PersonaSelectableList
+        hasPermission
+        multiSelect={false}
+        selectedPersonas={data}
+        onUpdate={mockUpdate}
+      />
+    );
+
+    expect(screen.getByTestId('popover')).toBeInTheDocument();
+    expect(screen.getByTestId('selectable-list')).toBeInTheDocument();
+    expect(screen.getByTestId('add-user')).toBeInTheDocument();
+  });
+
+  it('should show the persona list', async () => {
+    await act(async () => {
+      render(
+        <PersonaSelectableList
+          hasPermission
+          multiSelect={false}
+          selectedPersonas={data}
+          onUpdate={mockUpdate}
+        />
+      );
+    });
+
+    expect(screen.getByTestId('persona-list-item')).toBeInTheDocument();
+    expect(screen.getByText('username')).toBeInTheDocument();
+  });
+
+  it('should return null if hasPermission is false', async () => {
+    await act(async () => {
+      render(
+        <PersonaSelectableList
+          hasPermission={false}
+          multiSelect={false}
+          selectedPersonas={data}
+          onUpdate={mockUpdate}
+        />
+      );
+
+      expect(screen.queryByTestId('popover')).not.toBeInTheDocument();
+      expect(screen.queryByTestId('selectable-list')).not.toBeInTheDocument();
+    });
+  });
+
+  it('should call onUpdate function', async () => {
+    await act(async () => {
+      render(
+        <PersonaSelectableList
+          hasPermission
+          multiSelect={false}
+          selectedPersonas={data}
+          onUpdate={mockUpdate}
+        />
+      );
+      await act(async () => {
+        userEvent.click(await screen.findByTestId('update'));
+      });
+
+      expect(mockUpdate).toHaveBeenCalled();
```

Comment:
Instead of only checking if it's being called, we should check if it's been called with the expected value.